src/App.jsx

```diff
@@ -7,13 +7,14 @@ import Article from './Components/Article'
 import Profile from './Components/Profile'
 
 function App() {
+  const [topic, setTopic] = useState('')
```

Comment:
As the url is updated when filtering by topics, using useSearchParams in ArticlesList would be better than passing the states down